### ISSUE TITLE
About the training loss

### ISSUE THREAD

**Issue by hanssssssss** [xingmimfl/Deep3DFacereconstruction_pytorch3d#5]
Hi! I currently re-implemented the paper using pytorch like you. But I encounteered some problems. I used the pretrained resnet model from pytorch website. But the durning first 10 ecpoch of the training, the img-level loss is 0. I had no idea of thr reason.

This is the statistics of my training.
![e7874a86f45078483c819510235c564](https://user-images.githubusercontent.com/57829884/109652116-53709800-7b9a-11eb-9c4c-9f1b39abe50a.png)

Thanks
